guide/src/migration.md

```diff
@@ -110,8 +111,8 @@ let obj_ref = obj.borrow();
 ```
 
 #### Object extraction
-Now for `T: PyClass`, `&T` and `&mut T` don't have `FromPyObject` implementation.
-Instead, you can use `&PyCell`, `PyRef`, and `PyRefMut` for object extraction.
+`T: PyClass`, `&T` and `&mut T` no longer have `FromPyObject` implementations.
```

Comment:
```suggestion
For `PyClass` types `T`, `&T` and `&mut T` no longer have `FromPyObject` implementations.
```
To avoid confusion.  The current text sounds like `T` cannot be extracted, but I think it still can (as long as it's `Clone`)?

Reply:
> he current text sounds like T cannot be extracted, but I think it still can (as long as it's Clone)?

Ah yes, it should be also noted.